### PR TITLE
New Scarab Sprite

### PR DESCRIPTION
**Pull request by shazbot194**
scarabs2

Adds in a new scarab sprite. I was unable to test this just yet as I need to find a way to test them, but this should work. I will be happy to hear any way to test this. It also adds in the code to let these new sprites be used properly, which is the part that needs to be tested.

UPDATE!!!

So I got the new sprites done, sorry it took so long, I kept giving myself more and more to do, and only just now got to finalize these. I also was able to test the code, these do work from all my tests.

🆑 Shazbot
Changes: New Scarab sprite
/🆑

### CODE REVIEW

**Comment by Aurorablade** [comment]
i am gonna note these can look ALOT more biological then mechncial.

**Comment by shazbot194** [comment]
I would love to hear how you would like them to look. I tried my best to keep the same color scheme as the current guardians to make them more easily recognizable to the average player.

**Comment by Aurorablade** [comment]
they do not need to be the same color scheme at all, they can have the color accents sure! but they can look more alive/biological then a mechnical thing.  so go wild with that information then. They are more like parasitic biological things. Alien ones.

**Comment by shazbot194** [comment]
I'all look into making them a bit more organic in some way, possibly making those shell not a white-grey, but maybe something more nettle like, or xenomorph like.

**Comment by granodd** [comment]
Should make the shell a blackish color, or perhaps some other natural shade that a bug might have.

**Comment by taukausanake** [comment]
The side views look like some sweet af gamer chair. Scarab battle mount when?

**Comment by FalseIncarnate** [comment]
While the sprites are nice (definitely more organic like Fethas mentioned), I'm gonna nitpick that they only have 4 legs. Adding another pair of legs to bring this to 6 might be worth considering if room allows.

I'd also consider adjusting the side views by adding perhaps more of a "crown" to their heads, so it looks less like a exponential curve.

**Comment by shazbot194** [comment]
scarab_preview

Hey everyone, small update. I'm working with some people to improve the sprite even more. Some changes include changing the shell to a dull brown to make them more zerg/insect like, as well as some additions such as more eyes and hopefully more legs. This will take some time, so I ask this PR not be merged until I am finished with the sprites.

File: code/game/gamemodes/miniantags/guardian/guardian.dm [hunk 1]
```diff
@@ -356,18 +368,35 @@
...
+//			G.icon = 'icons/mob/mob.dmi'
+//			picked_name = pick("brood", "hive", "nest")
+//			to_chat(user, "[G.bio_fluff_string].")
+//			G.name = "[picked_name] swarm"
+//			G.color = picked_color
+//			G.real_name = "[picked_name] swarm"
+//			G.icon_living = "headcrab"
+//			G.icon_state = "headcrab"
+//			G.attacktext = "swarms"
+//			G.speak_emote = list("chitters")
```
Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/guardian/guardian.dm]: don't comment out. Remove, please.

Comment (shazbot194) [reply code/game/gamemodes/miniantags/guardian/guardian.dm]: Oh, oops, that was meant to be deleted, my appoligies, I will fix it when I get the sprites finished.

**Comment by shazbot194** [comment]
Hello all, so I got the new sprites done, these are improvements to my older ones. I was also able to test the code out a fair bit now that I can log in twice to a server, and as far as my testing went, there were no problems. So in summary, unless someone else has an idea for the sprites, this PR should be ready to merge!

**Comment by tigercat2000** [comment]
Seems alright.

**Pull request merged**